core/domain/suggestion_registry.py

```diff
@@ -2097,8 +2098,10 @@ def to_frontend_dict(
                 self.rejected_translation_word_count),
             'first_contribution_date': (
                 self.first_contribution_date.strftime('%b %d, %Y')),
-            'last_contributed_in_days': int(
-                (datetime.date.today() - self.last_contribution_date).days)
+            'first_contributed_in_days': utils.get_number_of_days_since_date(
```

Comment:
IIUC these changes are not needed in this PR. If I missed it and this field is used, please extract it into a separate PR. Ideally this PR should not touch anything on the backend side.

Reply:
removed the unnecessary code from the PR.